js/fetchData.js

```diff
@@ -0,0 +1,82 @@
+function createNode(element) {
+  return document.createElement(element);
+}
+
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+function hide(el) {
+  document.getElementById(el).style.display = "none";
+}
+
+function showBlock(el) {
+  document.getElementById(el).style.display = "block";
+}
+
+function showInline(el) {
+  document.getElementById(el).style.display = "inline";
+}
+
+const divMain = document.getElementById('questions');
+let url = 'http://localhost:5000/api/v1/questions?pages=1';
```

Comment:
`http://localhost:5000/api/v1/` can be stored in a `const` variable and be used by concatenating with other URL paths & parameters.